io.github.ra3xdh.qucs_s.yml

```diff
@@ -0,0 +1,48 @@
+id: io.github.ra3xdh.qucs_s
+runtime: org.kde.Platform
+runtime-version: 6.8
+sdk: org.kde.Sdk
+command: qucs-s
+rename-desktop-file: qucs-s.desktop
+rename-icon: qucs-s
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=host
+
+modules:
+  - name: dos2unix
+    buildsystem: simple
+    build-commands:
+      - make prefix=$FLATPAK_DEST install
```

Comment:
use no-autogen: true, make-args and/or make-install-args. Drop `buildsystem: simple`